packages/web-pkg/src/components/AppBanner.vue

```diff
@@ -55,6 +55,9 @@ export default defineComponent({
     const store = useStore()
     const router = useRouter()
     const appBannerSettings = unref(store.getters.configuration.currentTheme.appBanner)
+    const useAppBanner = computed(
+      () => appBannerSettings && Object.keys(appBannerSettings).length != 0
```

Comment:
Only checking for length could still mean someone gets a broken app banner since they configure one of the keys (e.g. title) but none of the others?

Reply:
For now we just want a way to disable the feature, because the current state of the implementation is not ready for a public release. Heavy changes expected afterwards. No need to go any extra miles here